<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>heroWindow</class>
 <widget class="QMainWindow" name="heroWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hero Kalk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nameLabel1">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hero</string>
    </property>
   </widget>
   <widget class="QLabel" name="HPLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
   </widget>
   <widget class="QLabel" name="MPLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MP</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill1Label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill2Label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill3Label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill4Label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heroLine1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="HPLine1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="MPLine1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill1Line1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill2Line1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill3Line1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill4Line1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fightButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>FIGHT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heroLine2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="HPLine2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill3Line2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill4Line2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="MPLine2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill1Line2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill2Line2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="skill">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel2">
    <property name="geometry">
     <rect>
      <x>683</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hero</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill2Label2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill4Label2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>350</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="MPLabel2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MP</string>
    </property>
   </widget>
   <widget class="QLabel" name="HPLabel2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill3Label2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectHero1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Hero 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectHero2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Hero 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="displayRes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Select or create an Hero</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="changeKalkHero">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ChangeKalk</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>439</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HERO POOL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="HList">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dpsH1But">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="htpH1But">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit per Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dptH1But">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Damage per Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="htpH1But_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit per Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dptH2But">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Damage per Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dpsH2But">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>heroLine1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>HPLine1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MPLine1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill1Line1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill2Line1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill3Line1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill4Line1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>heroLine2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>HPLine2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MPLine2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill1Line2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill2Line2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill3Line2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>skill4Line2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>displayRes</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>heroWindow</receiver>
   <slot>onHItemClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onHItemClicked(QListWidgetItem*)</slot>
 </slots>
</ui>
